bears/cmake/CMakeLintBear.py

```diff
@@ -21,10 +23,17 @@ class CMakeLintBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file,
+                         max_line_length: int=79,
                          cmakelint_config: path=''):
         """
+        :param max_line_length:
+             Maximum number of characters for a line.
++            When set to 0 allows infinite line length.
         :param cmakelint_config: The location of the cmakelintrc config file.
         """
+        if not max_line_length:
+            max_line_length = sys.maxsize
```

Comment:
you are not adding the setting to the args

Reply:
Still not done.

Reply:
Still not really done - wasnt tested.